microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -40,9 +48,9 @@ For information about temporarily pointing your app to a preview channel instead
 
 When you use a WebView2 SDK _release_ package, use the WebView2 Evergreen _Runtime_ on your development client, rather than a Microsoft Edge preview channel.  By default, a WebView2 app targets the Runtime rather than Microsoft Edge.  By design, the Microsoft Edge Stable channel doesn't support WebView2.
 
-The SDK _release_ package contains all of the stable Win32 C/C++ and .NET APIs, and doesn't include method signatures for experimental APIs.  All of the APIs that are in an SDK release package are fully supported, in an equal or higher build number of the WebView2 Runtime.
+The SDK _release_ package contains all of the Stable Win32 C/C++ and .NET APIs, and doesn't include method signatures for Experimental APIs.  All of the APIs that are in a Release SDK package are fully supported, in an equal or higher build number of the WebView2 Runtime.
```

Comment:
@liminzhu -- are there some Stable APIs that are in a Prerelease SDK but not in a Release SDK?  
re: "all the stable APIs" or "all the Stable-status APIs"

Reply:
APIs in production release are only a subset of Stable APIs in Prerelease. So I believe the answer is yes. 
Shall we say "The SDK _release_ package contains all of the Stable Win32 C/C++ and .NET APIs **in production release**"

Reply:
I'm going to push a commit that doesn't add new terminology / nomenclature such as "Prerelease-stable" or "production release", but just clarifies & tightens use of terms "Experimental, Stable, Release".

Reply:
Update: trying this recomm. from JP: 
try to only describe APIs as "stable" and "experimental", and only describe SDKs/packages as "release" and "prerelease"

Reply:
updated wording:
The Release SDK package contains all of the Stable Win32 C/C++ and .NET APIs that are in production release, and doesn't include method signatures for Experimental APIs. All of the APIs that are in a Release SDK package are fully supported, in an equal or higher build number of the WebView2 Runtime.